services/src/test/java/io/scalecube/services/ServiceConfigTest.java

```diff
@@ -13,17 +13,19 @@
   @Test
   public void test_services_config_port_increment_false() {
 
-    Microservices node = Microservices.builder().clusterConfig(
-        ClusterConfig.builder()
-
-            .transportConfig(TransportConfig.builder()
-                .port(3000)
-                .portAutoIncrement(false)
-                .build())) // cluster config
-
+    Microservices node = Microservices.builder()
+        .serviceTransport(TransportConfig.builder()
+            .port(3000)
+            .portAutoIncrement(false)
+            .build())
+        .clusterConfig(ClusterConfig.builder()
+            .port(4000)
+            .portAutoIncrement(false)) // cluster config
```

Comment:
actually this is out of the scope for this PR, but it seems to me that it is a bit unclear in terms of API that you need to pass here non built ClusterConfig and few lines above TransportConfig which is there built... this may be pretty confusing for someone who try to config this stuff. So consider to simplify this approach.